api/tests/devcomp/unit/infrastructure/datasources/learning-content/validation/module-schema.js

```diff
@@ -16,6 +16,25 @@ import { textElementSchema } from './element/text-schema.js';
 import { videoElementSchema } from './element/video-schema.js';
 import { htmlNotAllowedSchema, htmlSchema, uuidSchema } from './utils.js';
 
+const WHITELIST_ELEMENTS_SCHEMA = [
```

Comment:
À la réflexion, je trouve plus clair de dire :

```suggestion
const ALLOWED_ELEMENTS_SCHEMAS = [
```